hq/app/schedule/IamMessages.scala

```diff
@@ -22,8 +22,8 @@ object IamMessages {
   object VulnerableCredentials {
     private def message(account: AwsAccount): String = {
       s"Please check the following permanent credentials in AWS Account ${account.name}, " +
-      s"which have been flagged as either needing to be rotated or requiring multi-factor authentication (if you're " +
-      s"already planning on doing this, please ignore this message). If this is not rectified before the deadline, " +
+      s"which have been flagged as needing to be rotated (if you're already planning on doing this," +
```

Comment:
Hopefully we'll delete this file and move the relevant code to live with the other notifications soon!